torchvision/utils.py

```diff
@@ -575,3 +545,52 @@ def _make_ntuple(x: Any, n: int) -> Tuple[Any, ...]:
     if isinstance(x, collections.abc.Iterable):
         return tuple(x)
     return tuple(repeat(x, n))
+
+
+def _generate_color_palette(num_objects: int):
```

Comment:
Nit: could you put before `_log_api_usage_once` again to avoid unnecessary diff? If you like, you can also move `_parse_colors` there to "group" both functions together.

Reply:
applied in [658b98d](https://github.com/pytorch/vision/pull/7520/commits/658b98d31fe49aa87c4fb1953c309fef28d58bda)